repository/s.json

```diff
@@ -4843,6 +4843,16 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/djsegal/STGitKraken",
```

Comment:
"ST" is still just short for Sublime Text and redundant in a listing for packages for Sublime Text.

If you want to keep the reference to Sublime Text in the repo name, you can override a package's name with the `name` key here.